### ISSUE TITLE
VSCode Extension 0.0.25 still not working

### ISSUE THREAD

**Issue by tomschrot** [AvaloniaUI/AvaloniaVSCode#74]
Hi,
thank you for the new 0.0.25 release. It is getting better - slowly.

So far, the axaml code completion works!!! Thank you!

The previewer shows up just for a fraction of a second, then disappears again
the background grid is not shown

I am using DataGrid control, this may cause problems?

**Comment by prashantvc** [comment]
That's rather odd, I am using one of your sample code repo, it seem to work!
Do you seen any errors in the output window?

<img width="1912" alt="Screenshot 2023-11-29 at 12 41 46" src="https://github.com/AvaloniaUI/AvaloniaVSCode/assets/328076/3ac130e0-6e05-470c-aa4b-1bb7ad56b40a">

**Comment by tomschrot** [comment]
Hi

I did some more investigations.

I have 2 computers, 1 desktop and 1 notebook. On both machines there is VSCode installed. I use settings syncing via GitHub, thus both instances have the same configuration. Yesterday, I completely uninstalled VSC on my notebook, even deleting the remaining folders in the users directories. (Did I mention, that I have VSC installed system wide...)

Then I reinstalled VSC and after syncing everything is the way it should be. Opening the project folder, (restore &) building the project and run the app, all is fine. In VSC I place the cursor inside a tag neither code completion nor axaml preview pops up. I enclose the avalonia extension log which looks ok to me. 

The same procedure on the desktop machine does show code completion but no xaml preview!

So, VSC configuration should not be the problem. I suspect .NET config / Roslyn communication protocol to be the point...?
Maybe some paths missing?
[Avalonia Client.zip](https://github.com/AvaloniaUI/AvaloniaVSCode/files/13498038/Avalonia.Client.zip)

**Comment by tomschrot** [comment]
p.s. I set the extension log to verbose, maybe that helps

[Avalonia Client.zip](https://github.com/AvaloniaUI/AvaloniaVSCode/files/13498144/Avalonia.Client.zip)

**Comment by prashantvc** [comment]
Thanks for the sample, I will take a look.

> The same procedure on the desktop machine does show code completion but no xaml preview!
Sometimes the previewer service start up slow, maybe I can put some delay before previewer windows shows up

Ok, I just hit similar issue when switching between Mac and Windows. Could you make sure .NET path is correctly set in the VSCode?

You could also tell VS Code to ignore the setting sync for `dotnetAcquisitionExtension.existingDotnetPath`

``` json
  "settingsSync.ignoredSettings": [
    "dotnetAcquisitionExtension.existingDotnetPath"
  ],
```

**Comment by tomschrot** [comment]
Good Morning
"dotnetAcquisitionExtension.existingDotnetPath" does not change anything....
Maybe we should compare settings & MS C# extension according Roslyn...?

**Comment by prashantvc** [comment]
fixed in v0.0.26
